packages/gitbook/src/components/PageBody/PageBody.tsx

```diff
@@ -12,6 +12,7 @@ import { TrackPageViewEvent } from '../Insights';
 import { PageFeedbackForm } from '../PageFeedback';
 import { CurrentPageProvider } from '../hooks/useCurrentPage';
 import { DateRelative } from '../primitives';
+import { SuspenseLoadedHint } from '../primitives/LoadingStateProvider';
```

Comment:
We can group both.

I'm not sure how we could add more lint rules for imports; feel free to ignore my comments on this as the code already has plenty of messy imports